README.md

```diff
@@ -91,6 +93,10 @@ If everything is ok, finalize with:
 
 `now alias`
 
+## Contributing
+
+Back Your Stack is open source by default, and we welcome new contributions. Take a look at the [Contributing guidelines](Contributing.md) for ways to contribute.
```

Comment:
Spelling is "BackYourStack".

Not sure what "open source by default" means, "BackYourStack is open source" might be enough?

Reply:
Changed the spelling, and tightened. Good suggestion!